### PR TITLE
Create kiwiirc.subdomain.conf.sample

### PR DESCRIPTION
**Pull request by muskIRC**
add the kiwiirc irc client with some cache settings + letsencrypt certs

### CODE REVIEW

File: kiwiirc.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    listen [::]:80;
+    server_name kiwi.domain.com;
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name kiwi.domain.com;
```
Comment (Roxedus) [inline kiwiirc.subdomain.conf.sample:13]: This should be a a wildcard domain.

File: kiwiirc.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    add_header X-Content-Type-Options nosniff;
+    add_header X-XSS-Protection "1; mode=block";
+    expires 1y;
+    client_body_timeout 5s;
+    client_header_timeout 5s;
+    location / {
+        index index.html;
+        limit_conn addr 1;
+        proxy_cache my_cache;
+        root /usr/share/kiwiirc_master/;
```
Comment (Roxedus) [inline kiwiirc.subdomain.conf.sample:31]: This repository is reverse proxying, not hosting applications.

File: kiwiirc.subdomain.conf.sample [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+     }
+        location /static/config.json {
+        alias /etc/kiwiirc/client.json;
+    }
+    location /plugins/ {
+        alias /usr/share/kiwiirc_plugins/;
+    }
+    location /webirc/ {
+        # Forward incoming requests to local webircgateway socket
+        proxy_pass http://unix:/tmp/kiwiirc.sock;
```
Comment (Roxedus) [inline kiwiirc.subdomain.conf.sample:51]: Need to conform to the standard
```nginx
        set $upstream_app kiwiirc;
        set $upstream_port 1234;
        set $upstream_proto http;
        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
```

File: kiwiirc.subdomain.conf.sample [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        proxy_cache_bypass $cookie_nocache $arg_nocache;
+	    proxy_max_temp_file_size 0;
+        # Set http version and headers
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-Host   $host;
+        proxy_set_header X-Forwarded-Proto  $scheme;
+        proxy_set_header X-Forwarded-For    $remote_addr;
+        # Allow upgrades to websockets
+        proxy_set_header Upgrade     $http_upgrade;
+        proxy_set_header Connection  "upgrade";
```
Comment (Roxedus) [inline kiwiirc.subdomain.conf.sample:69]: Covered with proxy.conf

**Comment by drizuid** [comment]
OP didn't address the identified issues, closing.

**Pull request closed without merging**